starter-code/index.js

```diff
@@ -1,10 +1,37 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function(items, length) {
+  this.items = [];
+  this.length = this.items.length;
```

Comment:
they wanted `this.length = 0`, instead of having it hardcoded too `this.length = this.items.length;`